packages/blade/src/components/Alert/_decisions/decisions.md

```diff
@@ -0,0 +1,162 @@
+
+## API
+
+Annotated component with props:
+
+![Alert breakdown](./alert-breakdown.png)
+
+Sample usage:
+
+```jsx
+import { Alert, Link } from '@razorpay/blade';
+
+<Alert title="International Payments Only">
+  <Alert.Description>
+    Currently you can only accept payments in international currencies using PayPal. You cannot
+    accept payments in INR (₹) using PayPal. <Link href="https://razorpay.com">Know More</Link>
+  </Alert.Description>
+  <Alert.PrimaryAction
+    onClick={() => {
+      // do something
+    }}
+  >
+    Primary Action
+  </Alert.PrimaryAction>
+  <Alert.SecondaryAction
+    onClick={() => {
+      // do something
+    }}
+    href="https://razorpay.com"
+  >
+    Link
+  </Alert.SecondaryAction>
+</Alert>;
+```
+
+### Alert
+
+We'll expose an `Alert` component with the following API:
+
+| Prop          | Type                                     | Default     | Description                                                                                | Required |
+| ------------- | ---------------------------------------- | ----------- | ------------------------------------------------------------------------------------------ | -------- |
+| `children`    | `string`, Alert sub components           | `undefined` | Body content                                                                               | ✅       |
+| title         | `string`                                 | `undefined` | A brief heading                                                                            |          |
+| isDismissable | `boolean`                                | `true`      | Shows a dismiss button                                                                     |          |
+| contrast      | `high`, `low`                            | `low`       | Can be set to `high` for more prominent look _(not related to a11y)_                       |          |
+| isFullWidth   | `boolean`                                | `false`     | Spans the entire width of container, otherwise max width is restricted to 584px by default |          |
+| variant       | `info`, `positive`, `notice`, `negative` | `info`      | Sets the color tone of entire alert. Icon is set automatically based on variant \*         |          |
+| isBorderless  | `boolean`                                | `false`     | Removes borders and border radii, useful for creating a full bleed layout                  |          |
+
+We'll also expose the following sub components on `Alert` which can be passed as children to `Alert`. The order of passing these as children doesn't matter:
+
+| Sub component           | Description                                                          | Required |
+| ----------------------- | -------------------------------------------------------------------- | -------- |
+| `Alert.Description`     | For setting body content                                             | ✅       |
```

Comment:
What is the logic behind having `Description` as a compound component as compared to `Title` being a prop? There can be certain content structures where the title may get longer than the description.

Reply:
The design guidelines specify title should be concise and scannable (few words). Detailed description should come below that. This was the rationale behind making description a child (since it can render any JSX) and title (which only renders a string) as a prop